mifospay/src/main/java/org/mifos/mobilewallet/mifospay/data/firebase/api/services/MifosPayInstanceIDService.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2016 Google Inc. All Rights Reserved.
- *
+ * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
+ * <p>
```

Comment:
Are these necessary or you added them by mistake?

Reply:
@arboledasaa remove the unnecessary lines.